### ISSUE TITLE
WP-API REST Filtering

### ISSUE THREAD

**Issue by pcfreak30** [pods-framework/pods#3509]
The following is a crude class I have made for enabling pods to be searched over api: https://gist.github.com/pcfreak30/2cfb1d1040f6a29cb7f222bacb7cad35

using the following hook:

```
add_action( 'init', function () {
    PodsRESTHandlers::post_type_rest_support( 'my_post_type', 'my_post_type', 'WP_REST_Pods_Controller' );
}, 13 );
```

Should be a decent starting point for expanding pods rest support.

**Comment by pcfreak30** [comment]
Updated gist to hopefully fix #3412 as I could not update it either. Code is assuming object is a `Pods` not a `WP_Post`. My hack basically fixes that temporarily until the core code gets fixed :). Has to set it to a temp $id variable as php doesn't like setting one property to itself with different casing apparently.

**Comment by quasel** [comment]
REST support is now part of the pods core
